ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -234,7 +234,7 @@ private String location(ImportTableDesc tblDesc, Database parentDb)
 
     LOG.debug("adding dependent CopyWork/AddPart/MoveWork for table "
             + table.getCompleteName() + " with source location: "
-            + dataPath.toString() + " and target location " + tmpPath.toString());
+            + dataPath.toString() + " and target location " + tgtPath.toString());
 
     Task<?> copyTask = ReplCopyTask.getLoadCopyTask(replicationSpec, dataPath, tmpPath, context.hiveConf,
```

Comment:
Why not avoid MoveTask in LoadTable flow as well?

Reply:
check if we can remove ignore type

Reply:
added some comment for ignore type